src/otserv.cpp

```diff
@@ -213,6 +213,8 @@ int main(int argc, char* argv[]) {
 
 	g_dispatcher().start();
 	g_scheduler().start();
+	g_webhook().init();
```

Comment:
With the new thread pools this can go away.

Reply:
This one I don't think so, it initializes important information from the webhook and not necessarily from the thread.